styles.css

```diff
@@ -10,16 +33,19 @@ body {
     min-height: 100%;
 }
 
-#bandname {
-    color: white;
+.bandname {
+    color: #e0e0e0;
     font-size: 72px;
     font-family: Futura;
     font-weight: lighter;
     text-transform: uppercase;
-    letter-spacing: 12px;
-    position: absolute;
-    top: 40%;
-    left: 50%;
-    margin-right: -50%;
-    transform: translate(-50%, -50%)
-}
\ No newline at end of file
+    letter-spacing: 12px;}
+
+.position {
+  position: absolute;
+  text-align: center;
+  width: 95%;
+  padding: 5%;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);}
```

Comment:
Closing bracket should be in the new line.

Reply:
Fixed.